DetectorDescription/Detectors/src/SimpleCone_geo.cxx

```diff
@@ -0,0 +1,51 @@
+// $Id: $
+//====================================================================
+//  AIDA Detector description implementation for FCC
+//-------------------------------------------------------------------- 
+// Simple cone using dimensions to be used to define cone composed of 1 single material
+//
+// @author Clement Helsens
+//====================================================================
+#include "DD4hep/DetFactoryHelper.h"
+
+using namespace std;
+using namespace DD4hep;
+using namespace DD4hep::Geometry;
+
+
+static Ref_t create_element(LCDD& lcdd, xml_h e, SensitiveDetector sens)  {
```

Comment:
The second parameter gives a warning.Though it is generally needed for DD4hep, you can silence it by omitting the name (i.e. `(LCDD& lcdd, xml_h e, SensitiveDetector)`)